libraries/APM_OBC/APM_OBC.cpp

```diff
@@ -117,6 +117,30 @@ const AP_Param::GroupInfo APM_OBC::var_info[] = {
     // @User: Advanced
     AP_GROUPINFO("MAX_COM_LOSS", 14, APM_OBC, _max_comms_loss, 0),
 
+    // @Param: GEOFENCE
+    // @DisplayName: Enable geofence Advanced Failsafe
+    // @Description: This enables the geofence part of the AFS. Will only be in effect if AFS_ENABLE is also 1
+    // @User: Advanced
+    AP_GROUPINFO("GEOFENCE",     15, APM_OBC, _enable_geofence_fs, 1),
```

Comment:
Alright, sounds good. Tridge was the one who originally suggested I make these changes. I was thinking that making 1 'enabled' and 0 'disabled' made the most sense, but it'd be fine to change them to wording like "AFS_DISABLE_DUAL_LOSS" and the like to make the opposite true and have 0 keep the previous behavior.

I'm not too familar with the memory allocation, so hopefully Tridge will have some input on the 16 to 32 bit changeover - he was the one that suggested I do that, but it's entirely possible I did it wrong.

Thanks for the input!